### DISCUSSION TITLE
not displayed safetensors models

### DISCUSSION THREAD

**Discussion by abomonov** [Sanster/IOPaint#475] [Q&A] [answered]
When I add safetensors models downloaded on windows to cache\torch\hub, the models are not displayed in the IOPaint menu, what could be the problem? Thank you!

**Comment by Sanster** [comment, marked as answer]
If you are referring to the model of stable diffusion, you need to put the model in the following directory:
```
- cache_dir 
  |- stable_diffusion
      ├── sd-v1-5-inpainting.ckpt
      ├── sd-v1-5-inpainting.safetensors
      └── v1-5-pruned-emaonly.safetensors
  |- stable_diffusion_xl
      ├── sd_xl_base_1.0.safetensors
      └── sd_xl_base_1.0_inpainting_0.1.safetensors
```

**Reply by abomonov** [reply]
Thanks, I was able to solve the problem. I created a folder called stable_diffusion_xl in the cache directory and put the models there and everything worked.
